learning_wavelets/tests/models/exact_recon_unet_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import tensorflow as tf
+import tensorflow.keras.backend as K
+from tensorflow.keras.optimizers import Adam
+
+from learning_wavelets.models.exact_recon_unet import ExactReconUnet
+
+
+def test_init():
+    ExactReconUnet()
+    K.clear_session()
+
+def test_fit():
+    model=ExactReconUnet(n_output_channels=1, kernel_size=3, layers_n_channels=[4, 8, 16, 32])
```

Comment:
let's have a little bit less channels for the test
```suggestion
    model=ExactReconUnet(n_output_channels=1, kernel_size=3, layers_n_channels=[4, 8])
```